lms/djangoapps/support/views/contact_us.py

```diff
@@ -16,6 +17,9 @@ class ContactUsView(View):
     """
 
     def get(self, request):
+        if not configuration_helpers.get_value('CONTACT_US_PAGE', True):
```

Comment:
Just noticed that this wasn't updated on the upstream PR :cry: 
```suggestion
        if not configuration_helpers.get_value('CONTACT_US_ENABLE', True):
```